### DISCUSSION TITLE
How to edit locked properties in camoufox about:config

### DISCUSSION THREAD

**Discussion by wiseer2** [daijro/camoufox#47] [Q&A]
I have successfully adjusted these preferences when launching the browser through the Camoufox.exe directly, but when using the Python API, the settings seem to be overridden or locked. Is there any way to unlock or modify these preferences within the Camoufox API, or could there be something in the library source code that is enforcing these settings?

I have already tried passing prefs in the launch_options to disable these settings, but they remain locked:

python
firefox_user_prefs = {
    "privacy.clearOnShutdown.cookies": False,
    "privacy.clearOnShutdown.cache": False,
    "privacy.sanitize.sanitizeOnShutdown": False,
}
Any guidance or suggestions on how to manage these preferences programmatically using the Camoufox API would be greatly appreciated!

Thanks in advance!
![Capture](https://github.com/user-attachments/assets/5fad5c7a-8d7c-4a12-9077-c50853f68787)

**Comment by daijro** [comment]
Hello,

These config properties have been unlocked in v130.0.1-beta.13.

**Reply by wiseer2** [reply]
thank you so much
